server/backend/migration/20231003000000.go

```diff
@@ -0,0 +1,193 @@
+			if errRating != nil {
+				return errRating
+			}
+			errDish := addEntriesForCronJob(tx, "dishNameDownload", 302400)
+			if errDish != nil {
+				return errDish
+			}
+			return nil
+		},
+	}
+}
+
+func addEntriesForCronJob(tx *gorm.DB, cronName string, interval int32) error {
+	return tx.Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&model.Crontab{
+		Interval: interval,
+		Type:     null.StringFrom(cronName),
+		LastRun:  0,
+	}).Error
+}
+
+/*
+Updates the list of dishtags.
+If a tag with the exact german and english name does not exist yet, it will be created.
+Old tags won't be removed to prevent problems with foreign keys.
+*/
+func setNameTagOptions(db *gorm.DB) {
+	tagsNames := generateNameTagListFromFile("static_data/dishNameTags.json")
+	for _, v := range tagsNames.MultiLanguageNameTags {
+		var parentId int64
+
+		parent := model.DishNameTagOption{
+			DE: v.TagNameGerman,
+			EN: v.TagNameEnglish,
+		}
+
+		if err := db.Model(&model.DishNameTagOption{}).Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&parent).Error; err != nil {
+			fields := log.Fields{"en": v.TagNameEnglish, "de": v.TagNameGerman}
+			log.WithError(err).WithFields(fields).Error("Error while creating tag")
+		}
+		parentId = parent.DishNameTagOption
+
+		addCanBeIncluded(parentId, db, v)
+		addNotIncluded(parentId, db, v)
+	}
+}
+
+func addNotIncluded(parentId int64, db *gorm.DB, v nameTag) {
+	for _, expression := range v.NotIncluded {
+		fields := log.Fields{"expression": expression, "parentId": parentId}
+		err := db.Model(&model.DishNameTagOptionExcluded{}).
+			Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).
+			Create(&model.DishNameTagOptionExcluded{
+				Expression: expression,
+				NameTagID:  parentId}).Error
+		if err != nil {
+			log.WithError(err).WithFields(fields).Error("Unable to create new can be excluded tag")
+		}
+	}
+}
+
+func addCanBeIncluded(parentId int64, db *gorm.DB, v nameTag) {
+	for _, expression := range v.CanBeIncluded {
+		fields := log.Fields{"expression": expression, "parentId": parentId}
+
+		err := db.Model(&model.DishNameTagOptionIncluded{}).
+			Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).
+			Create(&model.DishNameTagOptionIncluded{
+				Expression: expression,
+				NameTagID:  parentId,
+			}).Error
+		if err != nil {
+			log.WithError(err).WithFields(fields).Error("Unable to create new can be excluded tag")
+		}
+	}
+}
+
+/*
+Reads the json file at the given path and checks whether the values have already been inserted into the corresponding table.
+If an entry with the same German and English name exists, the entry won't be added.
+The TagType is used to identify the corresponding model
+*/
+func setTagTable(path string, db *gorm.DB, tagType backend.ModelType) {
+	tagsDish := generateRatingTagListFromFile(path)
+	insertModel := getTagModel(tagType, db)
```

Comment:
I don't see why this line is extracted into another function.
This seems like unnessesary complexity